packages/editor/src/extensions/code-block/code-block.ts

```diff
@@ -276,7 +287,10 @@ export const CodeBlock = Node.create<CodeBlockOptions>({
             indentLength: options.amount
           });
           return true;
-        }
+        },
+      copy: (text) => () => {
+        return this.options.onClickToCopy(text);
+      }
```

Comment:
There is no need for a new `copy` command on the editor for abstracting the copy command to the clients because the `writeText` from `clipboard-polyfill` works on all platforms automatically.

Reply:
Yes, that'd be best. The editor gets rendered inside a webview so clipboard-polyfill works there as well.